delivery.yaml

```diff
@@ -0,0 +1,30 @@
+build_meta:
+  env:
+    LEIN_ROOT: true
+build_steps:
+- desc: Install dependencies
+  cmd: |
+    # OpenJDK
+    apt-get update
+    apt-get install -y openjdk-8*
+    # Leiningen
+    curl https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein >/usr/bin/lein
+    chmod a+x /usr/bin/lein
+    # Docker
+    curl -fsSL https://delivery.cloud.zalando.com/utils/ensure-docker | sh
+- desc: Run unit tests
+  cmd: |
+    docker run -d -p 5432:5432 postgres
+    lein test
+- desc: Build and push docker image
+  cmd: |
+    lein do clean, uberjar
+    touch target/scm-source.json # TODO Remove this after removing it from Dockerfile
```

Comment:
why not remove it immediately and fix the Dockerfile?

Reply:
This is just so we can test everything before remove the GoCD pipeline.